model/clusters_mgmt/v1/azure_type.model

```diff
@@ -106,4 +106,9 @@ struct Azure {
 	// configuration of the Cluster's Node Pool's Nodes is performed.
 	// By default this is configured as Azure Load Balancer. This value is immutable.
 	NodesOutboundConnectivity AzureNodesOutboundConnectivity
+
+	// Etcd encryption configuration
+	// Currently etcd data encryption is only supported with customer managed keys.
+	// Creating a cluster with platform managed keys will result in a failure creating the cluster.
```

Comment:
The description is not correct.
The defaulting must be mentioned.

It should be something like:
```
	// Etcd encryption configuration.
        // If not specified, etcd data encryption with platform managed keys is performed
```

And if we want, we can also temporarily add:
```

	// Currently etcd data encryption is only supported with customer managed keys.
	// Creating a cluster with platform managed keys will result in a failure creating the cluster.
```

which would be removed once platform managed keys is actually introduced